cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -432,9 +431,10 @@ func CreateControllerContext(s *options.CMServer, rootClientBuilder, clientBuild
 	return ctx, nil
 }
 
-func StartControllers(ctx ControllerContext, startSATokenController InitFunc, controllers map[string]InitFunc) error {
+func StartControllers(ctx ControllerContext, rootClientBuilder controller.ControllerClientBuilder, controllers map[string]InitFunc) error {
```

Comment:
I really don't want leak a `rootClientBuilder` into `StartControllers`.  Also, forcing the init function shape, makes sure the the init code for the SAToken controller will look consistent to people.